ansible/ansible.cfg

```diff
@@ -1,5 +1,6 @@
 [defaults]
 ansible_managed = Ansible managed, do not edit directly
+role_path = /opt/ansible_roles
 
 # Added to allow HQ to be cloned as a non-privileged user in Ansible >2.0.
 allow_world_readable_tmpfiles = True
```

Comment:
This is what Ansible was silently doing up to now, and what you have to explicitly allow after Ansible 2.0. It seems OK to me, but it might be better to clone HQ and install pip requirements as root, and then change ownership to the cchq user afterwards. It seems the only affected tasks are ansible/roles/commcarehq/tasks/main.yml.